src/boot/stylesheets/actions.scss

```diff
@@ -57,7 +57,7 @@
 	}
 }
 
-.wpnc__action-link {
+.wpnc__main .wpnc__action-link {
```

Comment:
For CSS specificity, it's just counting. ids always beat classes, classes always beat elements. So say if you have a selector with 100 elements and a second with one class, the second selector will win.

eg `( ids, classes, elements )` with inline styles or `!important` overruling any other rules.

In Calypso we try for ( 0, 1, 0 ) and for the most part use Component based rules than global resets, to avoid specificity issues like the one here.

In this example `button.wpnc__action-link` is ( 0, 1, 1 ). 
whiele `.wpnc__main .wpnc__action-link` is ( 0, 2, 0 ).

Which means that the second rule here has a higher specificity. It's not a huge deal here, but I like to trim down specificity if we don't need it.